nixos/modules/services/mail/rspamd.nix

```diff
@@ -323,6 +372,7 @@ in
         value = {
           description = "Rspamd socket ${toString each.index} for worker ${each.name}";
           wantedBy = [ "sockets.target" ];
+          after = optional (each.name == "rspamd_proxy") "postfix.service";
```

Comment:
Shouldn't this be before? You want the socket to be started before postfix uses it.

Reply:
This should be not required when socket activation is used.

Reply:
> This should be not required when socket activation is used.
You are right this line is not required. But thought this line is at best redundant and in my opinion also wrong since you want the socket to be created before not after the service, there is some logic behind adding an explicit dependency between postfix and the socket. In situations where the socket and postfix has been manually stopped an explicit dependency makes systemd also start the socket when you start postfix.

But I think this would have been a better way to add the dependency:
```
systemd.services.postfix = mkIf cfg.postfix.enable {
  after = [ "rspamd-normal-1.socket" ];
  requires = [ "rspamd-normal-1.socket" ];
}
```